src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -39,14 +38,7 @@ public static function debug_data() {
 		 * Set up the array that holds all debug information.
 		 *
 		 * When iterating through the debug data, the ordering of the sections
-		 * occurs in insertion-order of the assignments into this array. Setting
-		 * up empty values here preserves that specific ordering, so it doesn't
-		 * depend on when inside this method each section is otherwise assigned.
-		 *
-		 * When all sections have been modularized, this will be the final single
-		 * assignment of the sections before filtering and none will be empty.
-		 *
-		 * @ticket 61648
+		 * occurs in insertion-order of the assignments into this array.
```

Comment:
I'm a bit unsure about dropping this altogether. You are absolutely right that this should no longer be needed.

On the other hand, at least a part could make sense to be kept, in order to make sure, future developers are aware that this needs to be the sole place for assignments.

How about:
```suggestion
		 * occurs in insertion-order of the assignments into this array.
		 *
		 * This is the single assignment of the sections before filtering. Null-entries will
		 * be automatically be removed.
```

Reply:
Makes sense, commited! Thanks!